TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/config/Constants.java

```diff
@@ -0,0 +1,16 @@
+package org.firstinspires.ftc.teamcode.api.config;
+
+public class Constants {
+    public static final int    THREADS          = 4;
+
+    // Color sensor configuration
+    public static final double PARK_RED_FUDGE   = 250;
+    public static final double PARK_GREEN_FUDGE = 150;
+    public static final double PARK_BLUE_FUDGE  = 150;
+
+    // Drive Adjustments
```

Comment:
We'll need to do some testing to refine these values.

Reply:
I created #12 for this, resolving.